pkg/commands/drift/drift.go

```diff
@@ -44,34 +44,40 @@ func Process(ctx context.Context, organizationID, bucketQuery, driftignoreFile s
 	if err != nil {
 		return nil, fmt.Errorf("failed to initialize assets client: %w", err)
 	}
+	w := worker.New[*worker.Void](3)
```

Comment:
We should make the concurrency configurable (can be a follow-up PR).

Reply:
Done